### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by Hodor-Fergus**
Update the game over message so people know how to play again!

### CODE REVIEW

File: index.html
Comment (Drol3z) [inline index.html:1]: Great

**Review by Hodor-Fergus — COMMENTED** [review]
Awesome i see the differences in the code.

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (Hodor-Fergus) [inline index.html]:
```suggestion
<h2 hidden>Game over, refresh to play again. Lets make this a console.log🧑‍💻 🤖!</h2>
```

**Comment by Hodor-Fergus** [comment]
Finish

**Pull request merged**